python/envs/ControlRL/ControlHVACEnv.py

```diff
@@ -30,52 +25,39 @@ class AllVar(gym.Env):
         'video.frames_per_second' : 30
     }
 
-    def __init__(self):
-        
-        self.max_in_change=cfg['max_in_change']  # for 6 setting this to .1 will achieve .5 cost (high)
-        self.dt=cfg['rate_change']
+    def __init__(self):       
+        self.max_in_change=cfg['max_in_change']  
         self.viewer = None         
         self.max_PUE=cfg['max_PUE']
```

Comment:
this is just another nit-picky-comment about spacing `self.max_PUE=cfg['max_PUE']` -> `self.max_PUE = cfg['max_PUE']`